hbase_storage/src/main/java/com/splicemachine/access/hbase/H10PartitionAdmin.java

```diff
@@ -422,6 +422,50 @@ public void markDropped(long conglomId, long txn) throws IOException {
         }
     }
 
+    @Override
+    public void enableTableReplication(String tableName) throws IOException {
+        TableName tn = tableInfoFactory.getTableInfo(tableName);
+        try {
+            HTableDescriptor htd = admin.getTableDescriptor(tn);
+            if (admin.isTableEnabled(tn)) {
+                admin.disableTable(tn);
+            }
+
+            for (HColumnDescriptor hcd : htd.getFamilies()) {
+                hcd.setScope(HConstants.REPLICATION_SCOPE_GLOBAL);
+            }
+            admin.modifyTable(tn, htd);
+            SpliceLogUtils.info(LOG, "enabled replication for table %s", tn);
+        }
+        finally {
+            if (tn != null && !admin.isTableEnabled(tn)) {
+                admin.enableTable(tn);
```

Comment:
What if the table was already disabled to begin with?  The act of enabling replication should not enable the table in that case, right?  Also, should we only allow enabling of replication on tables that are currently enabled in HBase, and error out other cases, or is it OK to enable these as well?